cfme/utils/appliance/console.py

```diff
@@ -50,7 +50,7 @@ def run_commands(self, commands, autoreturn=True, timeout=10, channel=None, outp
             result = ''
             try:
                 while True:
-                    result += channel.recv(1).decode("ascii")
+                    result += channel.recv(1).decode("ascii", "backslashreplace")
```

Comment:
Here fixed below error:
`UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 0: ordinal not in range(128)`

Reply:
Would like to see the entire module run in PRT to make sure this doesn't break anything .

Reply:
This is not resolved, it is just running the `test_appliance_console_restore_db_network_negative` test case